packages/test-helpers/src/events.js

```diff
@@ -1,13 +1,19 @@
-function getEvents({ logs = [] }, eventName) {
+const { decodeEvents } = require('./')
+
+function getEvents(receipt, eventName, { decodeForAbi } = {}) {
```

Comment:
Why this mix of parameters style?

Reply:
Not sure exactly what you meant, but I push truly optional parameters to the options object at the end.

`eventName` is absolutely necessary (and makes sense), but an ABI for decoding the raw logs is not (open to suggestions for the name though—`decodeForAbi` was the least bad example that came to mind!).

Reply:
Ok, that answers my question: the ones encapsulated in the object are supposed to be optional, while the other ones don’t. ;-)
In reality, all parameters are strictly speaking optional, right?

Reply:
Haha well... in theory the `eventName` filter is also optional, yes.

I was thinking that in most of the cases you'll want to filter by `eventName`, but with the `decodeForAbi` option, do you think you'd use this without specifying a specific `eventName`?

Reply:
Not sure, but what I was missing, and it’s clear now, is the fact that the ones inside the object are meant to be optional. Because not only `eventName`, but also `receipt` is technically optional (which wouldn’t make any sense). I see the point that, in case of having multiple optional values, you don’t need to keep the order and worry about the ones you don’t want to pass.

Reply:
Yes, docs will hopefully help explain the thinking here 🙇‍♂️!